tools/torchci/clickhouse_query_perf.py

```diff
@@ -233,8 +235,25 @@ def perf_compare(args: argparse.Namespace) -> None:
         table.field_names = ["Test", "Avg Time", "Avg Mem"]
     for i, (new, base) in enumerate(query_ids):
         avg_time, avg_bytes = get_avg_stats(new)
-        if base:
+        if args.base:
             old_avg_time, old_avg_bytes = get_avg_stats(base)
+
+            if avg_time is None or old_avg_time is None:
```

Comment:
Is old_avg ever None? It's just set above

Reply:
yes `get_avg_stats` now propagates None if `query_ids` is  `None` or empty